avm/res/cdn/profile/tests/e2e/defaults/main.test.bicep

```
@@ -9,7 +9,7 @@ metadata description = 'This instance deploys the module with the minimum set of
 
 @description('Optional. The name of the resource group to deploy for testing purposes.')
 @maxLength(90)
-param resourceGroupName string = 'dep-${namePrefix}-cdn.profiles-${serviceShort}-rg'
+param resourceGroupName string = 'dep-${namePrefix}-afd.defaults-${serviceShort}-rg'
```

```suggestion
param resourceGroupName string = 'dep-${namePrefix}-cdn.profiles-${serviceShort}-rg'
```
Same comment as before